gpkit/constraints/single_equation.py

```diff
@@ -0,0 +1,41 @@
+"Implements SingleEquationConstraint"
+from operator import le, ge, eq
+from ..small_scripts import try_str_without
+from ..repr_conventions import _str, _repr, _repr_latex_
+
+
+class SingleEquationConstraint(object):
```

Comment:
I'm wondering if this could possibly inherit from `ConstraintSet`? They both implement a bunch of common methods (`str_without`, `as_posyslt1`), but there's no formal connection between them.